### ISSUE TITLE
Update styling of Common Components and Blog cards

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital.gov.bc.ca#850]
#### Issue
In #811, the styling of the Communities cards was changed to improve accessibility. The same type of cards (ones whose content does not contain a hyperlink, where the whole card is clickable) are being used in two other places: for Common Components and Blogs. The same styling should be applied across all.

#### Acceptance Criteria
- [x] Common Components and Blog cards have same styling applied as Communities

**Comment by mark-a-wilson** [comment]
Fixed in #869
